src/components/image/image.ts

```diff
@@ -426,8 +426,9 @@ export class SbbImageElement extends LitElement {
 
   public override connectedCallback(): void {
     super.connectedCallback();
-    // Check if the current element is nested in an `<sbb-teaser-hero>` element.
-    this._variantTeaserHero = !!hostContext('sbb-teaser-hero', this);
+    // Check if the current element is nested in an `<sbb-teaser-hero>` element on in an `<sbb-teaser-paid>` element.
+    this._variantTeaser =
```

Comment:
using closest() would be enough probably

Reply:
I tried at first but it didn't seem to work, if you want I can take a deeper look at it

Reply:
Yes can you maybe find out why it's not working?

Reply:
For teaser-paid it works just fine, but it breaks in the cases where the teaser-hero has an inlined sbb-image (instead of a slotted one). In this case I could use closest for the teaser-paid and hostContext for the teaser-hero. What do you think about it?

Reply:
Yeah just take teaser paid with closest. thanks :)